corehq/apps/app_manager/tests/test_tasks.py

```diff
@@ -9,13 +13,33 @@
     refresh_data_dictionary_from_app,
 )
 from corehq.apps.app_manager.tests.app_factory import AppFactory
-from corehq.apps.app_manager.tests.test_apps import AppManagerTest
-from corehq.apps.app_manager.tests.util import get_simple_form, patch_validate_xform
+from corehq.apps.app_manager.tests.util import add_build, get_simple_form, patch_validate_xform
 from corehq.apps.app_manager.views.releases import make_app_build
+from corehq.apps.domain.shortcuts import create_domain
 
 
 @patch_validate_xform()
-class AppManagerTasksTest(AppManagerTest):
+class AppManagerTasksTest(TestCase):
+    file_path = ('data',)
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.build1 = {'version': '1.2.0', 'build_number': 7106}
+        cls.build2 = {'version': '2.7.0', 'build_number': 20655}
+
+        add_build(**cls.build1)
+        add_build(**cls.build2)
+
+        cls.domain = 'test-domain'
+        create_domain(cls.domain)
```

Comment:
These are couch objects, right? I think they should be deleted with `addClassCleanup` or something long those lines.

Reply:
yes, thank you - 7ccea027e94